packages/application-shell/src/components/suspended-route/protected-route.tsx

```diff
@@ -3,18 +3,16 @@ import { PageUnauthorized } from '@commercetools-frontend/application-components
 
 type TProtectedRouteProps = {
   condition: boolean;
+  fallback?: React.ReactNode;
 } & RouteProps;
 
 const ProtectedRoute: React.FC<TProtectedRouteProps> = ({
   condition,
+  fallback = <PageUnauthorized />,
   children,
   ...routeProps
 }) => {
-  return condition ? (
-    <Route {...routeProps}>{children}</Route>
-  ) : (
-    <PageUnauthorized />
-  );
+  return condition ? <Route {...routeProps}>{children}</Route> : <>fallback</>;
```

Comment:
Sorry to jump in here late but if the `condition` is falsy we would be rendering the text "fallback" instead of rendering the component, righ?